packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/nodejs/integrationsteps.md

```diff
@@ -1,121 +1,118 @@
+![Screenshot showing a page displaying the sign-in page with a 'Forgot password?' link where users can click to reset their password.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-on-screenshot-sign-in-nodejs.png)
 
 </div>
 
-Create a **Recover password** page that initiates the reset password flow. This page accepts the user's email address and has a **Next** button that starts the reset flow.
+### 2. Show the password recovery page
+
+Next, show the password recovery page to the user. Create a page that accepts the account's username and includes a **Next** button that starts the recovery flow.
 
-<div class="common-image-format">
+<div class="common-image-format bordered-image">
 
-![Displays the Reset Password page that has a field to input your password.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-reset-nodejs.png)
+![Screenshot showing a page displaying the reset password page that has a field to input your password.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-reset-nodejs.png)
 
 </div>
 
-After the user clicks **Next** to start the reset flow, the next step is to call `idx.recoverPassword` and pass in the email address captured from the form. This starts the password recovery flow and returns a status of `Idx.Status:PENDING` and a `nextStep` field with inputs for an authenticator type.
+### 3. Submit username
 
-The call is shown in the `recover-password.js` file of the SDK sample application.
+The user enters their username on the password recovery page and selects **Next** to start the recovery process.  At this time, call `OktaAuth.idx.recoverPassword()`, passing in the username.
 
-```JavaScript
-router.post('/recover-password', async (req, res, next) => {
-  const { username } = req.body;
-  const authClient = getAuthClient(req);
+```javascript
+try {
   const transaction = await authClient.idx.recoverPassword({ username });
   handleTransaction({ req, res, next, authClient, transaction });
-});
+} catch (error) {
+  next(error);
+}
 ```
 
-The `handleTransaction.js` page routes the user to a select authenticators page (`/select-authenticator` in this example) that the user needs to use to select an authentication type. In this case, the email factor is configured.
-
-```JavaScript
-// authenticator authenticate
-    case 'select-authenticator-authenticate':
-      redirect({
-        req, res, path: '/select-authenticator'
-      });
-      return true;
+### 4. Display a list of available authenticators
+
+`OktaAuth.idx.recoverPassword()` returns a response indicating that the next step is to select an authenticator. Specifically, `IdxTransaction` returns a `status` of `PENDING`, `nextStep.name` equal to `select-authenticator-authenticate`, and a `option` item with a `value` of `okta_email` in the `nextstep.inputs[n].options` array.
```

Comment:
A list might help here.
```suggestion
`OktaAuth.idx.recoverPassword()` returns a response indicating that the next step is to select an authenticator. Specifically, `IdxTransaction` returns:
* a `status` of `PENDING`
* a `nextStep.name` equal to `select-authenticator-authenticate`
* an `option` item with a `value` of `okta_email` in the `nextstep.inputs[n].options` array

```

Reply:
Great feedback, I'll use this list style for future guides.